tests/stress/km/stress_tests_km.cpp

```diff
@@ -1048,6 +1050,283 @@ _print_test_control_info(const test_control_info& test_control_info)
+            bind_program_name += std::to_string(index);
+
+            // Try to get a handle to the 'BindMonitor_Calleexxindex' program.
+            bpf_program* callee = bpf_object__find_program_by_name(object, bind_program_name.c_str());
+
+            if (callee == nullptr) {
+                LOG_VERBOSE("({}) - bpf_object__find_program_by_name() failed. errno: {}", bind_program_name, errno);
+                REQUIRE(callee != nullptr);
+            }
+            LOG_VERBOSE("({}) - bpf_object__find_program_by_name() success.", bind_program_name);
+
+            fd_t callee_fd = bpf_program__fd(callee);
+            if (callee_fd < 0) {
+                LOG_VERBOSE("({}) - bpf_program__fd() failed. errno: {}", bind_program_name, errno);
+                REQUIRE(callee_fd > 0);
+            }
+            LOG_VERBOSE("({}) - bpf_program__fd() for callee_fd:{} success.", bind_program_name, callee_fd);
+
+            uint32_t result = bpf_map_update_elem(prog_map_fd, &index, &callee_fd, 0);
+            if (result < 0) {
+                LOG_VERBOSE("({}) - bpf_map_update_elem() failed. errno: {}", bind_program_name, errno);
+                REQUIRE(result == ERROR_SUCCESS);
+            }
+            LOG_VERBOSE("({}) - bpf_map_update_elem() for callee_fd:{} success.", bind_program_name, callee_fd);
+
+        } catch (...) {
+            // No need to terminate. We don't care about user mode issues here.
+        }
+    }
+
+    return prog_map_fd;
+}
+
+static void
+_invoke_mt_bindmonitor_tailcall_thread_function(thread_context& context)
+{
+    // Test bind
+    SOCKET socket_handle;
+    SOCKADDR_STORAGE remote_endpoint{};
+
+    if (context.role == thread_role_type::MONITOR_IPV4) {
+        socket_handle = WSASocket(AF_INET, SOCK_STREAM, IPPROTO_TCP, nullptr, 0, 0);
+        remote_endpoint.ss_family = AF_INET;
+    } else {
+        socket_handle = WSASocket(AF_INET6, SOCK_STREAM, IPPROTO_TCP, nullptr, 0, 0);
+        remote_endpoint.ss_family = AF_INET6;
+    }
+    REQUIRE(socket_handle != INVALID_SOCKET);
+
+    INETADDR_SETANY(reinterpret_cast<PSOCKADDR>(&remote_endpoint));
+    uint16_t remote_port = SOCKET_TEST_PORT + static_cast<uint16_t>(context.thread_index);
+
+    if (context.role == thread_role_type::MONITOR_IPV4) {
+        (reinterpret_cast<PSOCKADDR_IN>(&remote_endpoint))->sin_port = htons(remote_port);
+    } else {
+        (reinterpret_cast<PSOCKADDR_IN6>(&remote_endpoint))->sin6_port = htons(remote_port);
+    }
+
+    REQUIRE(bind(socket_handle, (PSOCKADDR)&remote_endpoint, sizeof(remote_endpoint)) == 0);
+    LOG_VERBOSE("Thread[{}] bind success to port:{}", context.thread_index, remote_port);
+
+    // Close socket.
+    closesocket(socket_handle);
+}
+
+static std::pair<bpf_object_ptr, fd_t>
+_load_attach_tail_program(
+    const std::string& file_name,
+    uint32_t thread_index,
+    const ebpf_attach_type_t attach_type,
+    const std::string program_name,
+    const bpf_prog_type program_type)
```

Comment:
```suggestion
    bpf_prog_type program_type)
```
no need to use const on value types, only pointer types